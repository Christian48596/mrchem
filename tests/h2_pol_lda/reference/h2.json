{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            -1.3230795180675052
          ],
          "radius": 1.4928836384543585
        },
        {
          "center": [
            0.0,
            0.0,
            1.3230795180675052
          ],
          "radius": 1.4928836384543585
        }
      ],
      "cavity_width": 0.37794522492515403,
      "charge": 0,
      "coords": [
        {
          "atom": "h",
          "xyz": [
            0.0,
            0.0,
            -0.7001435291738478
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.0,
            0.0,
            0.7001435291738478
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": true,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 5,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "h2.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {
      "ext_el-0.000000": {
        "components": [
          {
            "initial_guess": {
              "file_chk_x": "checkpoint/X_rsp_0",
              "file_chk_y": "checkpoint/Y_rsp_0",
              "file_x_a": "initial_guess/X_a_rsp_0",
              "file_x_b": "initial_guess/X_b_rsp_0",
              "file_x_p": "initial_guess/X_p_rsp_0",
              "file_y_a": "initial_guess/Y_a_rsp_0",
              "file_y_b": "initial_guess/Y_b_rsp_0",
              "file_y_p": "initial_guess/Y_p_rsp_0",
              "prec": 0.001,
              "type": "none"
            }
          },
          {
            "initial_guess": {
              "file_chk_x": "checkpoint/X_rsp_1",
              "file_chk_y": "checkpoint/Y_rsp_1",
              "file_x_a": "initial_guess/X_a_rsp_1",
              "file_x_b": "initial_guess/X_b_rsp_1",
              "file_x_p": "initial_guess/X_p_rsp_1",
              "file_y_a": "initial_guess/Y_a_rsp_1",
              "file_y_b": "initial_guess/Y_b_rsp_1",
              "file_y_p": "initial_guess/Y_p_rsp_1",
              "prec": 0.001,
              "type": "none"
            }
          },
          {
            "initial_guess": {
              "file_chk_x": "checkpoint/X_rsp_2",
              "file_chk_y": "checkpoint/Y_rsp_2",
              "file_x_a": "initial_guess/X_a_rsp_2",
              "file_x_b": "initial_guess/X_b_rsp_2",
              "file_x_p": "initial_guess/X_p_rsp_2",
              "file_y_a": "initial_guess/Y_a_rsp_2",
              "file_y_b": "initial_guess/Y_b_rsp_2",
              "file_y_p": "initial_guess/Y_p_rsp_2",
              "prec": 0.001,
              "type": "none"
            },
            "rsp_solver": {
              "checkpoint": false,
              "file_chk_x": "checkpoint/X_rsp_2",
              "file_chk_y": "checkpoint/Y_rsp_2",
              "final_prec": 0.001,
              "helmholtz_prec": -1.0,
              "kain": 3,
              "max_iter": 10,
              "method": "DFT",
              "orbital_thrs": 0.01,
              "orth_prec": 1e-14,
              "property_thrs": -1.0,
              "start_prec": 0.001
            }
          }
        ],
        "dynamic": false,
        "fock_operator": {
          "coulomb_operator": {
            "poisson_prec": 0.001,
            "shared_memory": false
          },
          "exchange_operator": {
            "exchange_prec": -1.0,
            "poisson_prec": 0.001
          },
          "xc_operator": {
            "shared_memory": false,
            "xc_functional": {
              "cutoff": 0.0,
              "functionals": [
                {
                  "coef": 1.0,
                  "name": "lda"
                }
              ],
              "spin": true
            }
          }
        },
        "frequency": 0.0,
        "perturbation": {
          "operator": "h_e_dip",
          "r_O": [
            0.0,
            0.0,
            0.0
          ]
        },
        "properties": {
          "polarizability": {
            "pol-0.000000": {
              "frequency": 0.0,
              "operator": "h_e_dip",
              "precision": 0.001,
              "r_O": [
                0.0,
                0.0,
                0.0
              ]
            }
          }
        },
        "unperturbed": {
          "fock_operator": {
            "coulomb_operator": {
              "poisson_prec": 0.001,
              "shared_memory": false
            },
            "exchange_operator": {
              "exchange_prec": -1.0,
              "poisson_prec": 0.001
            },
            "kinetic_operator": {
              "derivative": "abgv_55"
            },
            "nuclear_operator": {
              "proj_prec": 0.001,
              "shared_memory": false,
              "smooth_prec": 0.001
            },
            "xc_operator": {
              "shared_memory": false,
              "xc_functional": {
                "cutoff": 0.0,
                "functionals": [
                  {
                    "coef": 1.0,
                    "name": "lda"
                  }
                ],
                "spin": true
              }
            }
          },
          "localize": false,
          "precision": 0.001
        }
      }
    },
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 0.001,
          "shared_memory": false,
          "smooth_prec": 0.001
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "lda"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT",
        "prec": 0.001,
        "restricted": false,
        "screen": -1.0,
        "type": "gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        2.544077373851441e-17
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 2.312553471509131e-05,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            0.0,
            0.0,
            2.312553471509131e-05
          ],
          "vector_el": [
            0.0,
            0.0,
            2.3125534725416384e-05
          ],
          "vector_nuc": [
            0.0,
            0.0,
            0.0
          ]
        }
      },
      "geometry": [
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            -0.7001435291738478
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            0.7001435291738478
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -0.3452838567659957,
          -0.3452838567659957
        ],
        "occupation": [
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "b"
        ],
        "sum_occupied": -0.6905677135319914
      },
      "polarizability": {
        "pol-0.000000": {
          "frequency": 0.0,
          "isotropic_average": null,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "tensor": [
            null,
            null,
            null,
            null,
            null,
            null,
            0.0,
            0.0,
            7.221995950231811
          ]
        }
      },
      "scf_energy": {
        "E_ee": 1.3283641982722074,
        "E_eext": 0.0,
        "E_el": -1.8186521945216734,
        "E_en": -3.5666318553478566,
        "E_kin": 1.0734859823683878,
        "E_next": 0.0,
        "E_nn": 0.7141392859689609,
        "E_nuc": 0.7141392859689609,
        "E_tot": -1.1045129085527123,
        "E_x": 0.0,
        "E_xc": -0.6538705198144119,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 1,
      "routine": "mrchem.x",
      "total_cores": 1,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": {
      "ext_el-0.000000": {
        "components": [
          null,
          null,
          {
            "rsp_solver": {
              "converged": true,
              "cycles": [
                {
                  "mo_residual": 2.0619328084869655,
                  "property_update": -5.523959171474688,
                  "symmetric_property": -5.523959171474688,
                  "wall_time": 3.446845759
                },
                {
                  "mo_residual": 0.4485064452358458,
                  "property_update": -1.600959525202314,
                  "symmetric_property": -7.124918696677002,
                  "wall_time": 5.885403337
                },
                {
                  "mo_residual": 0.0314683038359275,
                  "property_update": -0.08682996683593824,
                  "symmetric_property": -7.211748663512941,
                  "wall_time": 6.140263724
                },
                {
                  "mo_residual": 0.005594449300447548,
                  "property_update": -0.01024728671887054,
                  "symmetric_property": -7.221995950231811,
                  "wall_time": 6.490117432
                }
              ],
              "wall_time": 21.962687677
            }
          }
        ],
        "frequency": 0.0,
        "perturbation": "h_e_dip",
        "success": true
      }
    },
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 1.3283641982722074,
        "E_eext": 0.0,
        "E_el": -1.8186521945216734,
        "E_en": -3.5666318553478566,
        "E_kin": 1.0734859823683878,
        "E_next": 0.0,
        "E_nn": 0.7141392859689609,
        "E_nuc": 0.7141392859689609,
        "E_tot": -1.1045129085527123,
        "E_x": 0.0,
        "E_xc": -0.6538705198144119,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
