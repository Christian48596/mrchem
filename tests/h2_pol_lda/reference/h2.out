                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         rebase-zora                              ***
***         Git commit hash    faecb83984c1d010ec90-dirty               ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Thu Mar 10 17:22:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               1
 Total cores             :                                               1
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : OpenMP (1 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000       -0.700144
    1       H            :        0.000000        0.000000        0.700144
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Project GTO molecular orbitals
 Precision               : 1.00000e-03
 Screening               : -1.00000e+00 StdDev
 Restricted              : False
 MO alpha file           : initial_guess/mrchem.moa
 MO beta file            : initial_guess/mrchem.mob
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999974574e-01
    1    1    b          :                              9.999999974574e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  1.073485982368
 E-N energy              :            (au)                 -3.566631855348
 Coulomb energy          :            (au)                  1.328364198272
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                 -0.653870519814
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  0.714139285969
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -1.818652194522
 Nuclear energy          :            (au)                  0.714139285969
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.104512908553e+00
                         :      (kcal/mol)             -6.930923087496e+02
                         :        (kJ/mol)             -2.899898221690e+03
                         :            (eV)             -3.005532604732e+01
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                 -0.345283856766
    1    1    b          :            (au)                 -0.345283856766
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -0.690567713532
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***               Computing Linear Response Wavefunction                ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Zero guess
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                             -1.000000000000e+00
    1    1    b          :                             -1.000000000000e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Zero guess
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                             -1.000000000000e+00
    1    1    b          :                             -1.000000000000e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Zero guess
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                             -1.000000000000e+00
    1    1    b          :                             -1.000000000000e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize linear response orbitals
 Frequency               : Static field
 Perturbation            : h_E_dip[z]
 Method                  : DFT
 Checkpointing           : Off
 Max iterations          : 10
 KAIN solver             : 3
 Start precision         : 1.00000e-03
 Final precision         : 1.00000e-03
 Helmholtz precision     : Dynamic
 Property threshold      : Off
 Orbital threshold       : 1.00000e-02
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual       Symmetric property                Update
---------------------------------------------------------------------------
    0          1.000000e+00           0.000000000000          0.000000e+00
    1          2.061933e+00          -5.523959171475         -5.523959e+00
    2          4.485064e-01          -7.124918696677         -1.600960e+00
    3          3.146830e-02          -7.211748663513         -8.682997e-02
    4          5.594449e-03          -7.221995950232         -1.024729e-02
---------------------------------------------------------------------------
                      SCF converged in 4 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000       -0.700144
    1       H            :        0.000000        0.000000        0.700144
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  1.073485982368
 E-N energy              :            (au)                 -3.566631855348
 Coulomb energy          :            (au)                  1.328364198272
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                 -0.653870519814
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  0.714139285969
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -1.818652194522
 Nuclear energy          :            (au)                  0.714139285969
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.104512908553e+00
                         :      (kcal/mol)             -6.930923087496e+02
                         :        (kJ/mol)             -2.899898221690e+03
                         :            (eV)             -3.005532604732e+01
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                 -0.345283856766
    1    1    b          :            (au)                 -0.345283856766
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -0.690567713532
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000       -0.000000        0.000023
 Magnitude               :            (au)                        0.000023
                         :         (Debye)                        0.000059
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000       -0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :        0.000000        0.000000        0.000023
 Magnitude               :            (au)                        0.000023
                         :         (Debye)                        0.000059
===========================================================================


===========================================================================
                       Polarizability (pol-0.000000)
---------------------------------------------------------------------------
 Frequency               :            (au)                        0.000000
                         :          (cm-1)                        0.000000
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Total tensor            :             N/A             N/A             N/A
                         :             N/A             N/A             N/A
                         :        0.000000        0.000000        7.221996
 Isotropic average       :            (au)                             N/A
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m 25s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
